torch/csrc/jit/passes/guard_elimination.cpp

```diff
@@ -247,6 +247,15 @@ struct GuardElimination {
     case aten::erf:
     case aten::erfc:
       return checkInputs(n, no_exceptions);
+    case aten::slice:
```

Comment:
why aren't we checking that `n->input(0)` isn't `isSummarized()`?

Reply:
I don't know what that does.

Reply:
Done